pkg/pb/task/task.pb.go

```diff
@@ -87,13 +87,13 @@ func (ResultCode) EnumDescriptor() ([]byte, []int) {
 // TaskMetadata is a task metadata abstraction that can be scheduled for execution at any CN node.
 type TaskMetadata struct {
 	// ID task id, global unique
-	ID string `protobuf:"bytes,1,opt,name=ID,json=iD,proto3" json:"ID,omitempty"`
```

Comment:
seams you don't change task/timestamp proto file,  you can remove these pb files from commit

Reply:
ok